docsite/components/alert.js

```diff
@@ -0,0 +1,8 @@
+// alert
+
+var removeBtn = document.querySelector(".remove-alert-btn");
```

Comment:
I would suggest to use the component with ids to perform js actions as in the bigger project if there are multiple same components or if the 'remove button' is used anywhere else, this will affect multiple components.

Reply:
Will take care of that  👍